site/app/templates/officeHoursQueue/QueueHeader.twig

```diff
@@ -3,18 +3,38 @@
     <p>{{viewer.getQueueMessage() | markdown}}</p>
   </div>
 {% endif %}
+
 <div class="content">
   <h1>
     Office Hours Queue
   </h1>
-  {% if not viewer.isGrader() %}
-    <p>
-      For more information:
+  <p>
+    For more information:
+    {% if not viewer.isGrader() %}
       <a target="_blank" href="https://submitty.org/student/queue">
         Getting Help through the Office Hours Queue
         <i style="font-style:normal;" class="fa-question-circle"></i>
       </a>
-    </p>
-    {% include 'officeHoursQueue/QueueStatus.twig' %}
+    {% else %}
+      <a target="_blank" href="https://submitty.org/grader/queue">
+        Managing the Office Hours Queue
+        <i style="font-style:normal;" class="fa-question-circle"></i>
+      </a>
+    {% endif %}
     <br>
+    <br>
+  </p>
+  {% if viewer.getQueueAnnouncementMessage() | length > 0 %}
+    <h3>Todays Announcements:</h3>
```

Comment:
I think the announcement header should rather be **h1** to match "Today's History" for consistency purposes, the header along with the paragraph containing the announcement should rather be in a div with some css as a background indicating that this is the announcement as this is how it looks in a student page, it is not separated from what follows.

![image](https://user-images.githubusercontent.com/30902406/86837731-9b88fa80-c09f-11ea-95c5-9830167acaf9.png)

Reply:
I added a background. I actually made the "today's announcement" a h2 not a h1 as well as switched the "today's history" also to h2. I am pretty sure a page is only supposed to have 1 h1 (in this case I have "Office Hours Queue"), so the "today's history" being a h1 is my bad to begin with.
![image](https://user-images.githubusercontent.com/18558130/86853831-49d37680-c085-11ea-95f1-e5247f8ec6c0.png)